### DISCUSSION TITLE
Optimization suggestions for the ternary operator

### DISCUSSION THREAD

**Discussion by BubbleGarten** [antonmedv/expr#692] [General]
After evaluating the expression following the question mark, there is no need to evaluate the expression following the colon. This is because I believe that such an overall expression is more intuitive:

<img width="1624" alt="image" src="https://github.com/user-attachments/assets/288148fa-f90f-493b-a409-7297718f313e">

**Comment by antonmedv** [comment]
I think this is a playground error. it makes subscription array -  strict. I need to update playground rules what if array is empty any fields are allowed.
